src/graph/Vertex.java

```diff
@@ -0,0 +1,74 @@
+package graph;
+
+import java.util.*;
+
+public class Vertex {
+
+  private int id;
+  private Map<Integer, Edge> edges;
```

Comment:
Add comments where possible. Here, you may want to say the Map is indiced with the edge's destination node id.